governance/votes/Executive vote - November 19, 2021.md

```diff
@@ -0,0 +1,180 @@
+
+- Increase maximum acquisition price from $1.5 million to $2.85 million.
+- Include Arkansas as a possible state.
+
+For more information, please see this [signal request thread].
+
+### Offboarding Vaults
+
+The following vaults will be offboarded by implementing the following changes if this executive proposal passes. Further context can be found [here](https://forum.makerdao.com/t/proposed-offboarding-collateral-parameters-2/11548).
+
+#### BAL-A
+
+As per [this poll](https://vote.makerdao.com/polling/QmcwtUau), the following parameters changes will occur to offboard BAL-A if this executive proposal passes:
+
+- Lower the [Liquidation Penalty (`chop`)](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%.
+- Lower the [Debt Ceiling (`line`)](https://makerdao.world/en/learn/governance/param-debt-ceiling) to 0 DAI.
+- Increase the [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) from 165% to 2000% over 30 days using dss-lerp.
+
+#### COMP-A
+
+As per [this poll](https://vote.makerdao.com/polling/QmRDeGCn), the following parameters changes will occur to offboard COMP-A if this executive proposal passes:
+
+- Lower the [Liquidation Penalty (`chop`)](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%.
+- Lower the [Debt Ceiling (`line`)](https://makerdao.world/en/learn/governance/param-debt-ceiling) to 0 DAI.
+- Increase the [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) from 165% to 2300% over 30 days using dss-lerp.
+
+#### AAVE-A
+
+As per [this poll](https://vote.makerdao.com/polling/QmPdvqZg), the following parameters changes will occur to offboard AAVE-A if this executive proposal passes:
+
+- Lower the [Liquidation Penalty (`chop`)](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%.
+- Lower the [Debt Ceiling (`line`)](https://makerdao.world/en/learn/governance/param-debt-ceiling) to 0 DAI.
+- Increase the [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) from 165% to 2100% over 30 days using dss-lerp.
+
+#### LRC-A
+
+As per [this signal poll](https://forum.makerdao.com/t/signal-request-offboard-unprofitable-collateral-types/8628), the following parameters changes will occur to offboard LRC-A if this executive proposal passes:
+
+- The previous dss-lerp for increasing the [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) of this collateral will be canceled.
+- A new dss-lerp contract will be deployed to increase the [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) to 24300% over 30 days.
+
+#### BAT-A
+
+As per [this signal poll](https://forum.makerdao.com/t/signal-request-offboard-unprofitable-collateral-types/8628), the following parameters changes will occur to offboard BAT-A if this executive proposal passes:
```

Comment:
https://vote.makerdao.com/executive/template-executive-vote-offboarding-vaults-mkr-vesting-and-core-unit-budget-distribution-october-8-2021?network=mainnet#proposal-detail ?